tutorials/cloud-functions-async/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Asynchronous patterns for Cloud Functions
+description: Build a simple HTTP function API for long running job or tasks.
+author: ptone
+tags: Cloud Functions, Firestore, serverless
```

Comment:
Firestore -> Cloud Firestore, here and any other mentions

Reply:
My understanding that we can drop "Cloud" after initial occurrence?

I did add it to all occurrences in headings and initial use

Reply:
Hmm - looking at https://cloud.google.com/guides/style/branding-guidelines (internal link) seems like Cloud is now included in all "short names" as well  - so addressing this.